### PR TITLE
(fix) Bug Bash with Justin

### PR DESCRIPTION
**Pull request by mazenchami**
# Description

[Trello Card](152-welcome-screen) — #110 
[Trello Card](151-venues-update-the-sort-order) — #111 
[Trello Card](150-info-sponsors-nasty-header-should-be-removed) — #112 
[Trello Card](149-info-contact-changes-to-headlines) — #113 
[Trello Card](148-info-should-contact-be-removed-from-top-right) — #114 
[Trello Card](147-explore-open-in-maps-consistency) — #115 
[Trello Card](146-explore-listed-items-should-be-a-carousel) — #116 

Wrapping up a bunch of bugs that @justinhuskey found while running through the app. See each Issue for more details.

# Graphics

| iOS            | Android            |
| -------------- | ------------------ |
| <br/><br /><br /><br /><br/> | <br/><br /><br /><br /><br/> |

# Checklist:

- [x] I have done a thorough self-review of my code
- [ ] I have tested with a screen reader and font-scaling turned on and added necessary accessibility features
- [x] I have run tests and linter

### CODE REVIEW

**Comment by morganick** [comment]
It could be my environment, but I have to make the following to changes to get the app to load without errors:

Use optional chaining for the talk: (https://github.com/infinitered/ChainReactApp2023/blob/main/app/services/api/webflow-helpers.ts#L157)
```javascript
talkUrl: schedule.talk?.["talk-url"],
```
This appears to have been a typo.

I've also commented out the following line: (https://github.com/infinitered/ChainReactApp2023/blob/main/app/screens/ScheduleScreen/ScheduleScreen.tsx#L191)
```javascript
// To achieve better performance, specify the type based on the item
// getItemType={(item) => item.variant}
```

I haven't investigated this as to why yet.

**Comment by mazenchami** [comment]
> It could be my environment, but I have to make the following to changes to get the app to load without errors:
> 
> Use optional chaining for the talk: (https://github.com/infinitered/ChainReactApp2023/blob/main/app/services/api/webflow-helpers.ts#L157)
> 
> ```js
> talkUrl: schedule.talk?.["talk-url"],
> ```
> 
> This appears to have been a typo.
> 
> I've also commented out the following line: (https://github.com/infinitered/ChainReactApp2023/blob/main/app/screens/ScheduleScreen/ScheduleScreen.tsx#L191)
> 
> ```js
> // To achieve better performance, specify the type based on the item
> // getItemType={(item) => item.variant}
> ```
> 
> I haven't investigated this as to why yet.

@morganick yes, thanks for bringing this up. This was all part of the discussion we had with Jenna/Josh late Friday. Josh is currently tackling the issues in #122. I would like to get his PR and this one merged prior to our AM meeting or i'll just end up delaying the meeting.

File: app/screens/ExploreScreen/ExploreScreen.tsx [hunk 1]
```diff
@@ -109,7 +105,7 @@ export const ExploreScreen: FC<TabScreenProps<"Explore">> = () => {
           stickySectionHeadersEnabled={false}
           sections={sections}
           renderSectionHeader={({ section: { title, data } }) =>
-            data.length > 0 ? (
+            data[0].length > 0 ? (
```
Comment (morganick) [inline app/screens/ExploreScreen/ExploreScreen.tsx:108]: I'm guessing we're expecting more to be added to `data`?

Comment (mazenchami) [reply app/screens/ExploreScreen/ExploreScreen.tsx:108]: Thats correct. Based on the WebFlow structure, that is a possibility, but for now we're not using it. I wanted to keep it a possibility for the design team

File: app/components/carousel/carousel.types.ts [hunk 2]
```diff
@@ -34,7 +35,7 @@ export interface DynamicCarouselItem {
   subtitle: string
 }
 
-interface DynamicCarouselProps {
+export interface DynamicCarouselProps {
```
[the remark this answers is not in the document]
Comment (mazenchami) [reply app/components/carousel/carousel.types.ts]: fixed here: https://github.com/infinitered/ChainReactApp2023/pull/117/commits/f729e22045b6b50abeb198fe08d4e796a5c56eec

**Pull request merged**